src/platform/testing/e2e/cypress/support/form-tester/index.js

```diff
@@ -135,8 +135,8 @@ const addNewArrayItem = $form => {
  *
  * @param {string} pathname - The pathname of the page to run the page hook on.
  */
-const performPageActions = pathname => {
-  cy.axeCheck();
+const performPageActions = (pathname, _13647Exception = false) => {
+  cy.axeCheck('main', { _13647Exception });
```

Comment:
These changes to the form-tester can be removed once the apps can pass the stricter axe checks.